ohsome/helper.py

```diff
@@ -204,29 +204,34 @@ def format_bboxes(
 
 
 def format_bpolys(
-    bpolys: Union[gpd.GeoDataFrame, shapely.Polygon, shapely.MultiPolygon, str]
+    bpolys: Union[
+        gpd.GeoDataFrame, gpd.GeoSeries, shapely.Polygon, shapely.MultiPolygon, str
+    ]
 ) -> str:
     """
     Formats bpolys parameter to comply with ohsome API
     :param
     bpolys: Polygons given as geopandas.GeoDataFrame or string formatted as GeoJSON FeatureCollection.
```

Comment:
```suggestion
    bpolys: Polygons given as geopandas.GeoDataFrame, geopandas.GeoSeries, Shapely.Polygon or GeoJSON FeatureCollection as string.
```

In my opinion, this could also be removed due to duplicated information/documentation when taking type hints into account.

Reply:
(Sidenote: I wonder why JSON is only supported as string and not as dict)